lib/samples/find_closest_facility_from_point/find_closest_facility_from_point.dart

```diff
@@ -42,22 +42,29 @@ class _FindClosestFacilityFromPointState
   static final _incidentsLayerUri = Uri.parse(
     'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Incidents/FeatureServer/0',
   );
+
```

Comment:
There are several blank lines added to each of the files. I don't think we should add them as part of this PR.

Reply:
It's dart auto-formatter which inserts the blank lines.